### PR TITLE
10.0-ir_attachment_url-image_resize_fix

### PR DESCRIPTION
**Pull request by rbikbov**
When the "image_resize_image" function was called, they received the error "binascii.Error: decoding with base64 codec failed (Error: Incorrect padding)", since the value of the binary field is the URL, not the base_64 string.

### CODE REVIEW

File: ir_attachment_url/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+`1.1.6`
```
Comment (yelizariev) [inline ir_attachment_url/doc/changelog.rst:1]: No update in manifest file

File: ir_attachment_url/models/ir_http.py [hunk 2]
```diff
@@ -102,6 +102,8 @@ def binary_content(cls, xmlid=None, model='ir.attachment', id=None, field='datas
                         with open(module_resource_path, 'rb') as f:
                             content = base64.b64encode(f.read())
                         last_update = str(os.path.getmtime(module_resource_path))
+                        # lint error fix
+                        del last_update
```
Comment (yelizariev) [inline ir_attachment_url/models/ir_http.py]: :-1: You shall not assign variable instead

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: ir_attachment_url/doc/changelog.rst (hunk 1 above)
Comment (RafiZz) [reply ir_attachment_url/doc/changelog.rst]: Do I need to move the message line to 1.1.5 or delete it completely?

Comment (rbikbov) [reply ir_attachment_url/doc/changelog.rst:1]: Done.

File: ir_attachment_url/models/ir_http.py (hunk 2 above)
Comment (rbikbov) [reply ir_attachment_url/models/ir_http.py]: Done.

**Comment by yelizariev** [comment]
Still 

ir_attachment_url/models/ir_http.py:84:13: F841 local variable 'last_update' is assigned to but never used

**Pull request merged**